Sources/DistributionItem.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  DistributionItem.swift
+//  Relativity
+//
+//  Created by Dan Federman on 12/27/16.
+//  Copyright © 2016 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import CoreGraphics
+import Foundation
+import UIKit
+
+
+// MARK: – DistributionItem
+
+
+public enum DistributionItem {
+    
+    case fixed(CGFloatConvertible)
```

Comment:
Originally used `CGFloat`, but then I had to use `CGFloat(8.0)` anywhere I wanted an `8.0` space. Went with `CGFloatConvertible` to avoid excessive casting.